Dockerfile

```diff
@@ -8,8 +8,4 @@ RUN pip install --no-cache-dir --upgrade -r /usr/src/app/requirements.txt
 
 COPY ./app /usr/src/app
 
-ENTRYPOINT ["uvicorn", "app.main:app", "--proxy-headers", "--host", "0.0.0.0"]
-
-CMD ["--port", "8000"]
-
-
+ENTRYPOINT ["uvicorn", "app.main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
Mostly for simplicity, as to override `CMD` in the Dockerfile we would have to add a `command:` directive in `docker-compose.yml`, and then I was thinking we would have to code for extra environment variables for the ports (at this point it almost makes more sense for the user to create a new `yml`). 

That said, upon reflection, since the Docker image for the API will likely be pulled from Dockerhub in most cases, I think it makes sense to keep this configurability in and trust that if a user really wanted to customize all the ports (for the API or graph), they could modify the docker-compose file itself/create one that suits their needs (without having to change anything in the Dockerfile). So I've reversed this change without adding any additional environment variable substitutions in `docker-compose.yml`.